js/islandora_book_reader.js

```diff
@@ -16,7 +16,8 @@
     this.bookUrl = document.location.toString();
     this.imagesBaseURL = settings.imagesFolderUri;
     this.logoURL = '';
-    this.mode = settings.mode;
```

Comment:
This was missing, good. Still, it would be great to pass the `mode` also as a setting instead of solely depending on the `URL Fragment` processing ? Since https://github.com/Islandora/islandora_internet_archive_bookreader/pull/138/files#diff-542ab515026d028eec06ed5e93b11086R986 is setting params.mode, or maybe i got this wrong and the defaults are already setting that params?

Reply:
I agree. I have returned the mode as it is used elsewhere!